### PR TITLE
FS-3837 fix dpif letter case

### PR DESCRIPTION
**Pull request by RamuniN**
https://dluhcdigital.atlassian.net/browse/FS-3837

### Description
fixed DPIF sub-criteria letter case

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

File: app/blueprints/assessments/routes.py [hunk 1]
```diff
@@ -752,6 +752,16 @@ def activity_trail(application_id: str):
...
+    _ = [  # get sub_criteria name from id
+        score.update(
+            {
+                "sub_criteria_name": get_sub_criteria(
+                    application_id, score.get("sub_criteria_id")
+                ).name
+            }
+        )
+        for score in scores
+    ]
```
Comment (srh-sloan) [inline app/blueprints/assessments/routes.py:764]: Can we have a unit test to check the name is populated please?

Comment (RamuniN) [reply app/blueprints/assessments/routes.py:764]: Added the test. Can you look again please?

Comment (RamuniN) [reply app/blueprints/assessments/routes.py:764]: @srh-sloan

**Review by srh-sloan — APPROVED** [review]
👍

**Pull request merged**